test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -3083,6 +3083,24 @@ Expect.describe('Additional Auth Button', function () {
       expect(test.form.additionalAuthButton().hasClass('new-class')).toBe(true);
     });
   });
+  itp('displays custom button translation', function () {
```

Comment:
another test case when both title or i18nKey is passed, i18nKey will be ignored.